src/main/java/frc/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.subsystems.swerve;
+
+import com.pathplanner.lib.config.PIDConstants;
+import com.pathplanner.lib.controllers.PPHolonomicDriveController;
+import com.pathplanner.lib.path.PathConstraints;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.constants.MathConstants;
+
+public record SwerveConstants(
+	String logPath,
+	String stateLogPath,
+	String velocityLogPath,
+	double velocityAt12VoltsMetersPerSecond,
```

Comment:
Why exactly 12 volts? I think it'll be a bit more clear to pass it with proper SI and then multiply it.

Reply:
12 is the default battery voltage permitted in frc

Reply:
I think it should be clear from the name. And, those kinds of things could change in the future, hence breaks OCP.